shared/tracker/index.js

```diff
@@ -131,6 +131,7 @@ export default connect(
   },
   (stateProps, dispatchProps, ownProps) => {
     return {
+      proofs: [],
```

Comment:
Unclear. We can remove it if we feel like it makes no sense. I wasn't
trying to fix this bug with this ticket it was just stopping me from doing
the tickets fixes

On Thursday, March 3, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/tracker/index.js
> <https://github.com/keybase/client/pull/2204#discussion_r54974421>:
>
> > @@ -131,6 +131,7 @@ export default connect(
> >    },
> >    (stateProps, dispatchProps, ownProps) => {
> >      return {
> > +      proofs: [],
>
> Hmm I can't seem to repro. Sounds like this might be a bug in how we store
> the tracker state? Does the rest of the tracker state exists and it's only
> proof that's missing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2204/files#r54974421>.
>